### PR TITLE
Update github.md

### PR DESCRIPTION
**Pull request by fantasywind**
add default callback URL path

### CODE REVIEW

File: doc/integration/github.md [hunk 1]
```diff
@@ -18,6 +18,7 @@ GitHub will generate an application ID and secret key for you to use.
     - Homepage URL: The URL to your GitLab installation. 'https://gitlab.company.com'
     - Application description: Fill this in if you wish.
     - Authorization callback URL: 'https://gitlab.company.com/'
+    - If install from source, default callback URL is '${YOUR_DOMAIN}/import/github/callback'
```
Comment (stanhu) [inline doc/integration/github.md]: Doesn't this apply to omnibus installations as well?

Comment (fantasywind) [inline doc/integration/github.md]: I install from source only, cannot know about omnibus path structure, sor.
If structure is same with source installation, I can fix it!

Comment (axilleas) [inline doc/integration/github.md]: Yes, it's the same in Omnibus as well. Remove `If install from source` and we can merge :)

**Comment by axilleas** [comment]
Cool, thanks @fantasywind

**Pull request merged**